### PR TITLE
This implements feature request for bios logs #57

### PR DESCRIPTION
**Pull request by arunkoshy**
tested and ready for review.

### CODE REVIEW

File: ctrl/ctrl1.go [hunk 1]
```diff
@@ -531,6 +532,25 @@ func home(w http.ResponseWriter, r *http.Request) {
 		}
 		conn.Close()
 
+	case "rom_sol_log":
+		// We must get the username from the request
```
Comment (vejmarie) [inline ctrl/ctrl1.go:536]: Is that comment relevant here ?

File: ctrl/ctrl1.go [hunk 2]
```diff
@@ -531,6 +532,25 @@ func home(w http.ResponseWriter, r *http.Request) {
...
+			os.Truncate(solLogPath+"bios_sol.log", 0)
+		}
+
+		// we need to start also the console
+		sutIP := r.FormValue("bmcip")
+		base.Zlog.Infof("SUTIP:", sutIP)
+
+		var argsConsole []string
+		argsConsole = append(argsConsole, sutIP)
+		romSerialConsoleCmd = exec.Command("/usr/local/production/bin/bioslog", argsConsole...)
```
Comment (vejmarie) [inline ctrl/ctrl1.go:549]: Could the path be an input parameter of the microservices instead of an hard coded value ?

**Review by vejmarie — CHANGES_REQUESTED** [review]

File: ctrl/ctrl1.go (hunk 2 above)
Comment (arunkoshy) [reply ctrl/ctrl1.go:549]: taking discussion offline ;-)

File: ctrl/ctrl1.go (hunk 1 above)
Comment (arunkoshy) [reply ctrl/ctrl1.go:536]: can fix

**Comment by arunkoshy** [comment]
@vejmarie: addressed your feedback with the latest changes. we will refactor further as we retrofit the test framework.

**Review by susmithp — APPROVED** [review]
All looks good.

**Comment by arunkoshy** [comment]
thanks @susmithp :)

**Review by vejmarie — APPROVED** [review]

**Pull request merged**